src/cli.yaml

```diff
@@ -82,6 +82,28 @@ subcommands:
             about: Sets the output file to use
             required: true
             index: 2
+  - update:
+      about: >
+          Attempts to update an existing Valheim server installation. By
+          default this involves checking for an update, if an update is
+          available, the server will shut down, updated, and brought back online
```

Comment:
Thanks for your work!
I think it should be « the server will be shut down, updated, and brought back online » instead of « the server will shut down, updated, and brought back online »

Reply:
I believe you're right. Fixed in eb4a80e083e8a95212198a8dbeb87745ee2087e0!